backend/generator.js

```diff
@@ -10,10 +10,10 @@ exports.executeScript = function (socket, formData) {
     return false;
   }
   var email = formData.email;
-  var gitUrl = formData.gitUrl;
+  var gitUrl = "https://github.com/" + formData.username + "/" + formData.reponame + ".git";
```

Comment:
Will Travis handle that automatically? Cuz if not here, we'll have to form a URL inside the generate script (to clone the repo)